### ISSUE TITLE
Create a production release 

### ISSUE THREAD

**Issue by mezerotm** [mezerotm/cmv#41]
Create a production release so that we can link our [github.io](http://mezerotm.github.io/cmv/) to a stable release, currently at the moment our page changes as the master changes. I would just like to get people's opinion on weather we should keep it dynamic with how the project grows or if we want a more stable page.

**Comment by hardnett** [comment]
Good point. I had not been paying attention to this. I suggest we have a static page of the screen with Alpha coming soon or something like that. Then only have it change to the application once we start releasing. So while we work on the Beta release it will only show the alpha release. It will switch to the beta release once it's ready. 

Do we have any stats on accessing it? Has anyone been checking it out?

**Comment by mezerotm** [comment]
Usually when I introduce the cmv at meetups I go to. I like showing people our work and I do show them the page. If you want to get statistics on the URL if we use [google's url shortener](https://goo.gl/#) and shorten the link we can start tracking stats on the url.

**Comment by hardnett** [comment]
Thats why I like it for showing our current release i.e. alpha, beta, etc. I am just not sure if we want the incremental updates visible. 

I was wondering if we already had some data from github. Once we go alpha, I had planned to have the application deployed on a server like the one I already have and we can just add a domain name for $10 per year. Or we may use AWS, Google Cloud, Heroku, or something like that.  We could then use the services of the hosting company to track. The URL shortner is only useful when you know that people will only access the site by the shortened URL. I have used it in the past for situations where I am presenting something to a group and I want to track how many people from that specific group have accessed the material. In our case, the hosting services data would be more complete.
